milestone-3/frontend/src/components/Home.jsx

```diff
@@ -0,0 +1,22 @@
+import "../styles/Home.module.css";
+import RecipePreview from "./RecipePreview";
+import recipeData from "../recipeData";
+import { Outlet } from "react-router-dom";
+
+export default function Home() {
+  return (
+    <div>
+      <h2>welcome to my bakery!</h2>
+      <p>here are some recipes I like</p>
+      <h3>recipes</h3>
+      {recipeData.map(recipe => (
+        <RecipePreview
+          name={recipe.recipeName}
+          desc={recipe.recipeDescription}
+          img={recipe.recipeImage}
+        />
+      ))}
+      <Outlet />
```

Comment:
I haven't seen Outlet used before, but after some research (correct me if I'm wrong :D) it does not look like it is doing anything here.